wifi/test/u_wifi_captive_portal_test.c

```diff
@@ -128,7 +130,7 @@ static uDeviceCfg_t gDeviceCfg = {
     }
 };
 
-static int32_t gStartTimeMs = -1;
+static uTimeoutStop_t gTimeoutStop = {0};
```

Comment:
why is {0} here needed and not in all other cases maybe

Reply:
Mostly because it is not my code and this is what the code did before, i.e. a zero value for the `gStartTime` that went before was used to mean "no timeout is set".  However, you are right that, as far as I can tell, it shouldn't be required (`gTimeoutStop` and the `gStartTime` that went before, are populated before the `keepGoingCallback()` is given to anything).  Will remove the initialisation.